napari_aicssegmentation/widgets/workflow_step_widget.py

```diff
@@ -72,23 +72,22 @@ def _add_slider(
         if default_value < param.min_value or default_value > param.max_value:
             raise ValueError("Default value outside of min-max range")
 
-        # Build dictionary of keyword args for slider widgets
-        kwargs = dict()
-        kwargs["step"] = param.increment
-        kwargs["max"] = param.max_value
-        kwargs["min"] = param.min_value
-        kwargs["value"] = default_value
-
-        magicgui_widget = None
+        magicgui_slider = None
         if param.data_type == "float":
-            magicgui_widget = FloatSlider(**kwargs)
+            magicgui_slider = FloatSlider()
+            magicgui_slider.setDecimals(3)
```

Comment:
hmmmm how about this?
```
if param.increment <= 0.001:
    magicgui_slider.setDecimals(3)
```

Reply:
Actually better would be
`if param.increment < 0.01:`

Reply:
So if the increment is `0.002` we hide the third decimal? :smiley: 

Wrong code but I understand what you want to do.I figure we can get fancy with these possible little UI improvements later, if @jxchen01 wants? For now it doesn't seem crazy (and is consistent) to just display all float sliders with 3 decimals.

Reply:
Comments came out of order :x

Reply:
Haha yeah you caught me before the correction. Up to you